scripts/deployment/deploy_liquidity_hub.sh

```diff
@@ -182,7 +175,7 @@ function init_incentive_factory() {
 	# Instantiate the contract
 	code_id=$(jq -r '.contracts[] | select (.wasm == "incentive_factory.wasm") | .code_id' $output_file)
 	$BINARY tx wasm instantiate $code_id "$init" --from $deployer --label "White Whale Incentive Factory" $TXFLAG --admin $deployer_address
-
+	sleep $tx_delay
```

Comment:
added delays in here so there's time for the node to catch the tx so the next query doesn't fail